packages/flutter/test/material/input_date_picker_form_field_test.dart

```diff
@@ -378,4 +380,18 @@ void main() {
       expect(find.text(errorFormatText), findsOneWidget);
     });
   });
+
+  testWidgets('FocusNode can request focus', (WidgetTester tester) async {
+    final FocusNode focusNode = FocusNode();
+    await tester.pumpWidget(inputDatePickerField(
+      focusNode: focusNode,
+    ));
+    expect(focusNode.hasFocus, isFalse);
```

Comment:
Can you add a check that the focusNode in TextFormField is the same as the one that passes in?

Reply:
Sure, will this test be part of TextFormField or InputDatePickerFormField?

Reply:
Done 👍✅, could you please check?

Reply:
This looks good, thanks!